src/components/Venue.js

```diff
@@ -233,11 +233,23 @@ class Venue extends Component {
       return null
     }
 
-    return (
-      <div className="venues-overlay__img">
-        <img src={this.props.images[2].url} title={this.props.name} />
-      </div>
-    )
+    if (this.props.images[2].url) {
```

Comment:
Could we do and then return one `div`?
`const imageSource = this.props.images[2].url || this.props.image[6].url`